### PR TITLE
Remote Storage for Commerce Cloud additions

### PR DESCRIPTION
**Pull request by rvr31**
## Purpose of this pull request
This pull request is adding missing documentation of enabling remote storage for Adobe Commerce Cloud.

## Affected pages
https://experienceleague.adobe.com/docs/commerce-operations/configuration-guide/storage/remote-storage/cloud-support.html

### CODE REVIEW

File: help/configuration/remote-storage/cloud-support.md [hunk 1]
```diff
@@ -5,6 +5,50 @@ description: See guidance on how to set up remote storage for Adobe Commerce on
...
+}
+```
+
+To add the new environment variable for remote storage using the `magento-cloud` cli use the following command
+
+```bash
+magento-cloud variable:create --level environment --name REMOTE_STORAGE --json true --inheritable false --value '{"driver":"aws-s3","prefix":"uat","config":{"bucket":"aws-bucket-id","region":"eu-west-1","key":"optional-key","secret":"optional-secret"}}'
+```
+
+Running this command will create a new environment variable called REMOTE_STORAGE with the specified JSON configuration. Alternatively, you could use the web interface to add the variable to the appropriate environment.
```
Comment (shrielenee) [inline help/configuration/remote-storage/cloud-support.md]:
```suggestion
Running this command will create a new environment variable called `REMOTE_STORAGE` with the specified JSON configuration. Alternatively, you could use the web interface to add the variable to the appropriate environment.
```

File: help/configuration/remote-storage/cloud-support.md [hunk 2]
```diff
@@ -5,6 +5,50 @@ description: See guidance on how to set up remote storage for Adobe Commerce on
...
+During the deployment you should see the following line in your deployment logs `INFO: Remote storage driver set to: "aws-s3"`.
+
+### Optional authentication
+
+The `key` and `secret` are optional. During the creation of the variable, it is possible to set it to `sensitive` so that the value is not visible in the web interface hiding the `key` and `secret`. If you choose not to use the `key` and `secret` authentication method, you need to ensure the server is authorized to the S3 bucket by other means. In that case, the `key` and `secret` should be omitted from the JSON. 
+
+## Syncing the remote-storage
```
Comment (shrielenee) [inline help/configuration/remote-storage/cloud-support.md:42]:
```suggestion
## Syncing the remote-storage

```

**Review by shrielenee — DISMISSED** [review]

File: help/configuration/remote-storage/cloud-support.md [hunk 3]
```diff
@@ -5,6 +5,50 @@ description: See guidance on how to set up remote storage for Adobe Commerce on
...
+After enabling the remote store module make sure to syncronize the current media files to the remote store location.
```
Comment (meker12) [inline help/configuration/remote-storage/cloud-support.md]:
```suggestion
After enabling the remote store module make sure to synchronize the current media files to the remote store location. 
```

File: help/configuration/remote-storage/cloud-support.md [hunk 4]
```diff
@@ -5,6 +5,50 @@ description: See guidance on how to set up remote storage for Adobe Commerce on
...
+Running this command will create a new environment variable called REMOTE_STORAGE with the specified JSON configuration. Alternatively, you could use the web interface to add the variable to the appropriate environment.
+During the deployment you should see the following line in your deployment logs `INFO: Remote storage driver set to: "aws-s3"`.
+
+### Optional authentication
+
+The `key` and `secret` are optional. During the creation of the variable, it is possible to set it to `sensitive` so that the value is not visible in the web interface hiding the `key` and `secret`. If you choose not to use the `key` and `secret` authentication method, you need to ensure the server is authorized to the S3 bucket by other means. In that case, the `key` and `secret` should be omitted from the JSON. 
+
+## Syncing the remote-storage
+After enabling the remote store module make sure to syncronize the current media files to the remote store location. 
+To start the syncronisation run the command below on the enviroment where you enabled the remote storage.
```
Comment (meker12) [inline help/configuration/remote-storage/cloud-support.md]:
```suggestion
To start the synchronization run the command below on the environment where you enabled the remote storage.
```

**Review by meker12 — COMMENTED** [review]
Maybe need to double-check this info.

In the [Cloud release notes](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/release-notes/ece-tools-package.html), it specifically says that the REMOTE_STORAGE variable is not supported on Cloud infrastructure.

And the remote storage config instructions point to another [topic](https://experienceleague.adobe.com/docs/commerce-operations/configuration-guide/storage/remote-storage/cloud-support.html?lang=en) that explains how to configure remote storage using Fastly.

**Review by hguthrie — CHANGES_REQUESTED** [review]
This requires a technical review, so blocking for now.

**Comment by rvr31** [comment]
> Maybe need to double-check this info.
> 
> In the [Cloud release notes](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/release-notes/ece-tools-package.html), it specifically says that the REMOTE_STORAGE variable is not supported on Cloud infrastructure.
> 
> And the remote storage config instructions point to another [topic](https://experienceleague.adobe.com/docs/commerce-operations/configuration-guide/storage/remote-storage/cloud-support.html?lang=en) that explains how to configure remote storage using Fastly.

Hey!

I think the support part is about getting support from the cloud support team on your remote storage location. By enabling remote storage, you move the storage of media files to a private location. Thus the support team cannot access your media files.

If you could get someone from the product or cloud team to review, that be great. We implemented this on a customer's project and found a lack of documentation. Hence we added it.

**Comment by hguthrie** [comment]
@robin31  I am on the Cloud team. We are reviewing. Thanks!

**Review by hguthrie — APPROVED** [review]
Okay, I am importing this and will process it internally. Thanks for the PR!

**Pull request merged**